lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexWorkerManager.java

```diff
@@ -41,17 +45,46 @@
     this.reefTask = reefTask;
   }
 
+  /**
+   * Sends an {@link VortexAggregateFunction} and its {@link VortexFunction} to a
+   * {@link org.apache.reef.vortex.evaluator.VortexWorker}.
+   */
+  <TInput, TOutput> void sendAggregateFunction(final int aggregateFunctionId,
+                                               final VortexAggregateFunction<TOutput> aggregateFunction,
+                                               final VortexFunction<TInput, TOutput> function) {
+    final TaskletAggregationRequest<TInput, TOutput> taskletAggregationRequest =
+        new TaskletAggregationRequest<>(aggregateFunctionId, aggregateFunction, function);
+    vortexRequestor.send(reefTask, taskletAggregationRequest);
```

Comment:
`send()` is for guaranteeing that the message is sent before Tasklets are launched, right?
Adding a short comment why we want `send()` will be helpful to follow. :)